src/tests/test_demographics.py

```diff
@@ -25,7 +25,7 @@ def test_sex_distribution(default_module):
     count = 100000
     sex = default_module.initialize_sex(count)
     female = np.sum(sex == SexType.Female)
-    assert pytest.approx(female/count, rel=0.05) == FEMALE_RATIO
+    assert pytest.approx(female/count, rel=0.01) == FEMALE_RATIO
```

Comment:
I feel like we should decide as a group on a standard for deciding the thresholds of probabilistic tests? We seem to go back and forth on these quite a bit so it would be good to be consistent. If we can codify this somewhere (in some shared variable for example) that would be even better...

Reply:
That would be good and give some more reassurance that we are not just enabling the tests to pass. Can we come up with a single rule, given that we have several distributions and generative process?

Reply:
Followed up in #45.